providers/terraform/connection/hcl_manifest.go

```diff
@@ -51,6 +51,15 @@ func NewHclConnection(id uint32, asset *inventory.Asset) (*Connection, error) {
 
 func newHclConnection(path string, asset *inventory.Asset) (*Connection, error) {
 	// NOTE: right now we are only supporting to load either state, plan or hcl files but not at the same time
+	if len(asset.Connections) != 1 {
+		return nil, errors.New("only one connection is supported")
+	}
+
+	confOptions := asset.Connections[0].Options
+	includeTerraform := true
```

Comment:
nit: Should we rename this to `includeDotTerraform`? As it is now, it somehow reads strange to not include terraform in a terraform connection.

Reply:
Great idea